confgenerator/testdata/invalid/linux/metrics-receiver_unsupported_type/golden_error

```diff
@@ -1 +1 @@
-metrics receiver "receiver_1" with type "unsupported_type" is not supported. Supported metrics receiver types: [hostmetrics].
\ No newline at end of file
+metrics receiver "receiver_1" with type "unsupported_type" is not supported. Supported metrics receiver types: [hostmetrics, iis, mssql].
```

Comment:
On Linux, `iis` and `mssql` are not supported, right?

Reply:
Ah I see. This changed because we switched from collectd to otel, which has a TODO. Got it.